src/libraries/Common/tests/System/Net/Http/PostScenarioTest.cs

```diff
@@ -211,6 +242,12 @@ public async Task PostNonRewindableContentUsingAuth_PreAuthenticate_Success(Conf
         [Theory, MemberData(nameof(RemoteServersMemberData))]
         public async Task PostAsync_EmptyContent_ContentTypeHeaderNotSent(Configuration.Http.RemoteServer remoteServer)
         {
+            if (remoteServer.HttpVersion.Major >= 2 && PlatformDetection.IsBrowser)
```

Comment:
I think a different issue should be opened for it, if it is not the same failure as others in #53018, with details of the failure.

Reply:
I created https://github.com/dotnet/runtime/issues/53591 and https://github.com/dotnet/runtime/issues/53592, for cases which have more specific error message and tagged the relevant tests.